packages/gatsby/src/bootstrap/load-plugins/load-internal-plugins.ts

```diff
@@ -0,0 +1,154 @@
+import { slash } from "gatsby-core-utils"
```

Comment:
This file used to be `load-plugins/load` and imported like `import { loadPlugins as loadPluginsInternal } from "./load"`. Name is was changed to be explicit